src/Raven.Client/Constants.cs

```diff
@@ -261,6 +261,10 @@ private Metadata()
 
                 public const string Refresh = "@refresh";
 
+                public const string Archive = "@archive";
```

Comment:
Maybe `@archive-at` ? Not the same as `@expires` or `@refreshed` but I would like to avoid the similarity  to `@archived`
That is bound to be confusing. 
Or maybe change `@archived` to something else? Maybe `@is-archived` ?

Reply:
Tbh, @archive was also confusing for us, but as you mentioned we wanted to keep the similarity to @expires. We agree that @archive-at is far superior. In such a case, I believe that the contrast between @archive-at and @archived props will be adequate, and the names are still meaningful enough.

Reply:
Tbh, `@archive` was also confusing for us, but as you mentioned we wanted to keep it similar to `@expires` and other ones. I agree that `@archive-at` is far superior. In such a case, I believe that the contrast between `@archive-at` and `@archived` props will be adequate, and the names are still meaningful enough.

Reply:
Applied here - https://github.com/ravendb/ravendb/pull/16732/commits/bf48a3f65b31712ff189f643434cff80fdfb586b